README.md

```diff
@@ -72,3 +72,13 @@ For feedback related to this SDK please visit the
 [Nfield website].
 
 [Nfield website]: https://www.nipo.com/
+
+## Releases
+ - How to create and publish a new `-alpha` or `-beta` version. `-alpha` and `-beta` versions can only be published on a DevOps private/internal Nfield Feed:
+    - Create a new branch with the this patern : `*/ci-*` or merge a commit into `master`
+    - This branch will trigger a new build in DevOps pipelines
+    - After the build a `Release pipeline` will allow to Push the new nuget version in the `Nfield` internal feed. These versions __don't create__ `GitHub releases`.
+ - How to create and publish a new `release` on the Nuget.org public Feed
+    - Create a new release branch Example: `release/2023-P3S5`
+    - This branch will trigger a new build in DevOps pipelines
+    - After the build a `Release pipeline` will allow to Push the new nuget version in the `Nuget.org` feed and it will create the `GitHub release` repo.
```

Comment:
Are there any approvals in this process?

Reply:
Yes is using an [Environment](https://dev.azure.com/niposoftware/Nfield/_environments/104?view=deployments) that only OPS can approve